packages/cxl-ui/src/components/cxl-base-card.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import '@vaadin/details';
+import '@vaadin/button';
+import cxlBaseCardStyles from '../styles/cxl-base-card-css.js';
+
+export class CXLBaseCardElement extends LitElement {
+  static get styles() {
+    return [cxlBaseCardStyles];
+  }
+
+  separator = html`<span> | </span>`;
+
+  @state() _tagsHasChildren = false;
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) theme = 'Base';
+
+  @property({ type: String }) name = '';
+
+  @property({ type: String }) time = '';
+
+  @property({ type: String }) instructor = '';
+
+  @property({ type: String }) avatar = '';
+
+  @property({ type: Boolean, reflect: true }) new = false;
+
+  _slotHasChildren (e) {
+    const slot = e.target
+    const { name } = slot
+    const children = slot.assignedNodes()
+    this[`_${name}HasChildren`] = !!children.length
+  }
+
+  _renderHeader() {
+    return html`
+      <header>
+        <div class="info">
+          <div class="tags">
+            ${this.theme ? html`<span class="tag">${this.theme}</span>`: ''}
+            ${this.theme && this._tagsHasChildren ? this.separator : ''}
+            <slot name="tags" @slotchange=${this._slotHasChildren}></slot>
+            ${this.new ? html`${this.theme ? this.separator : ''}<span class="tag new">NEW</span>` : '' }
+          </div>
+          <div class="name">
+            ${this.name}
+          </div>
+          <div class="attributes">
+            <div class="time">
+              ${this.theme.toLowerCase() === 'Base' ? html`<vaadin-icon icon="lumo:clock"></vaadin-icon>` : ''}
```

Comment:
Won't this always resolve to false?

Reply:
Hum, it's almost like a developer might have selected all "course" occurrences in the file, substituted for "Base" and never looked back... Not my finest hour, but a nice example of why code review is effective, thank you for that.